adult_catalog.yaml

```diff
@@ -18,3 +18,38 @@ adult:
   immutable:
     - age
     - sex
+  feature_input_order_drop_first:
+    - age
+    - fnlwgt
+    - education-num
+    - capital-gain
+    - capital-loss
+    - hours-per-week
+    - workclass_Private
+    - marital-status_Non-Married
+    - occupation_Other
+    - relationship_Non-Husband
+    - race_White
+    - sex_Male
+    - native-country_US
+  feature_input_order_bin:
```

Comment:
Do we always need both or can we "standardize" on one type of encoding?

Reply:
I'm not sure how 😅 One of our main problems is, that we have to keep different ML models for binary and non-binary categorical features. 

If there is a common ground to loose these, this would be a major improvement.

Reply:
Ah yeah right. Remind me again, why this is the case that we have to have models for both? Because the Recourse Method expect them? Can we (easily) re-write them in this case?

Reply:
Some recourse methods only want one column for binary categories and some want to have two, and they're not flexible enough to handle both types. If we would be able to manipulate the original code, that all recourse methods expect every type of input, this would be a major breakthrough 😄 But this would take a lot of time, I think.

Hmm you mean, writing methods that transform the binary into the non-binary type?

Reply:
Ok makes sense, lets support the current case then 👍🏽